### PR TITLE
Multisig wallet deployment task.

### PR DESCRIPTION
**Pull request by H-Pavel**
# Checklist:

- [ ] Code follows the style guidelines
- [ ] Documentation updated
- [ ] Corner cases evaluated and handled
- [ ] index.js exports updated
- [ ] Automatic tests added/updated
- [ ] Automatic tests pass
- [ ] Sanity check passed
- [ ] Self-review passed
- [ ] Version updated

### CODE REVIEW

File: src/tasks/deploy_multisig.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  async execute(args) {
+    console.log('Deploying MultiSigWallet contract.');
+
+    const options = this.parseOptions(args);
+    if (options === null) {
+      return;
+    }
+
+    let approvalAdresses;
+    if (typeof config.multisigApprovalAddresses !== 'undefined') {
```
Comment (vanruch) [inline src/tasks/deploy_multisig.js]:
```suggestion
    if (config.multisigApprovalAddresses !== undefined) {
```

File: src/tasks/deploy_multisig.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      approvalAdresses = Array.from(config.multisigApprovalAddresses.split(','));
```
Comment (vanruch) [inline src/tasks/deploy_multisig.js]: Why do we need Array.from here?

File: src/tasks/deploy_multisig.js (hunk 1 above)
Comment (H-Pavel) [reply src/tasks/deploy_multisig.js]: Fixed

File: src/tasks/deploy_multisig.js (hunk 2 above)
Comment (H-Pavel) [reply src/tasks/deploy_multisig.js]: It was just convinient to see the type (array) at once. Nevertheless it works without Array.from.
Fixed
